packages/automerge-repo-storage-indexeddb/src/index.ts

```diff
@@ -0,0 +1,134 @@
+import { StorageAdapter } from "@automerge/automerge-repo"
+
+export class IndexedDBStorageAdapter extends StorageAdapter {
+  private dbPromise: Promise<IDBDatabase>
+
+  constructor() {
+    super()
+    this.dbPromise = new Promise((resolve, reject) => {
+      const request = indexedDB.open("automerge", 1)
+
+      request.onerror = function () {
+        reject("Database error: " + request.error)
```

Comment:
Can you just reject(request.error)? 
It seems to be just a standard Error object https://developer.mozilla.org/en-US/docs/Web/API/IDBRequest/error